nixos/modules/services/misc/airsonic.nix

```diff
@@ -38,9 +38,6 @@ in {
         default = "127.0.0.1";
         description = ''
           The host name or IP address on which to bind Airsonic.
-          Only relevant if you have multiple network interfaces and want
```

Comment:
I might still leave some of this, but worded differently:

```
The host name or IP address on which to bind Airsonic.
The default value is appropriate for first launch, when the
default credentials are easy to guess. It is also appropriate
if you intend to use the virtualhost option in the service
module. In other cases, you may want to change this to a
specific IP or 0.0.0.0 to listen on all interfaces.
```

But feel free to reword it as you think reasonable.

Reply:
Added your quote: https://github.com/NixOS/nixpkgs/pull/158421/commits/00a0c8ba0978dfb3e6af13899a2015bfddadc29e

Reply:
Cool, LGTM